build/config/PublishData.json

```diff
@@ -0,0 +1,36 @@
+{
+  "master": {
+    "nugetPush": [
+        {
+            "kind": "PerBuildPreRelease", 
```

Comment:
But what's the process ? We're not shipping *every* release for every package we produce (since we're still producing multiple builds per day for insertions). I would assume we would have left this to PerBuildPreRelease and then still one-off shipped the release bits.

Is your thinking that once we've transitioned to shipping mode we're always doing that, even for intermediate builds?